utils/tests/utils.go

```diff
@@ -41,3 +43,17 @@ func RenamePath(oldPath, newPath string, t *testing.T) {
 		t.FailNow()
 	}
 }
+
+func SetJfrogHome(homePath string) {
+	if err := os.Setenv(coreutils.HomeDir, homePath); err != nil {
+		log.Error(err)
+		os.Exit(1)
+	}
+}
+
+func CleanUnitTestsJfrogHome(homePath string) {
+	os.RemoveAll(homePath)
```

Comment:
Let's not ignore the error returned by ```os.RemoveAll```.

Reply:
Fixed.